content/documentation/install-npm.mdx

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Also I think the "install a node" link in the first paragraph of the getting started page is broken since we moved this page. There might be others too 😬

Reply:
yeah a lot of links are broken, trying to get buy-in now on the structure before i go fix everything